drivers/gpio/Kconfig.pca95xx

```diff
@@ -15,3 +15,9 @@ config GPIO_PCA95XX_INIT_PRIORITY
 	depends on GPIO_PCA95XX
 	help
 	  Device driver initialization priority.
+
+config PCA95XX_INTERRUPT_SUPPORT
```

Comment:
Must be `GPIO_PCA95XX_INTERRUPT_SUPPORT` to keep the subsystem prefix associated with the Kconfig variable group.  `GPIO_PCA95XX_INTERRUPT` is shorter and consistent with existing SX1509B driver.